src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,26 @@
+package duke.task;
+public class Deadline extends Todo {
+    String deadline;
+
+    public Deadline(String name, String deadline) {
+        super(name);
+        setDeadline(deadline);
+    }
+
+
+    public String getDeadline() {
+
+        return deadline;
+    }
+
+    public void setDeadline(String deadline) {
+
+        this.deadline = deadline;
+    }
+
+    @Override
+    public String toString() {
+        String boolString = super.getIsDone() ? "X" : " ";
```

Comment:
Avoid magic strings like "X" - consider extracting them out as constants and name them using proper naming conventions